orchagent/port.h

```diff
@@ -53,17 +68,17 @@ class Port
     Type                m_type;
     int                 m_index = 0;    // PHY_PORT: index
     int                 m_ifindex = 0;
+    sai_uint32_t        m_mtu;
```

Comment:
this is irrelevant to the pr. consider to remove.

Reply:
done.